integration-tools/src/test/java/uk/ac/bbsrc/tgac/miso/integration/test/BoxScanTests.java

```diff
@@ -21,7 +22,7 @@ public abstract class BoxScanTests<T extends BoxScan> {
    *  <LI>B02: 44444</LI>
    * </OL>
    */
-  protected abstract T getFullScan();
+  protected abstract T getFullScan() throws JsonProcessingException;
```

Comment:
This is another reason not to store data as JSON in `DP5MirageScan`. Once we have the scan object, we should already know that it's valid and shouldn't need to worry about `JsonProcessingException`s